docs/root/configuration/http/http_filters/grpc_stats_filter.rst

```diff
@@ -17,6 +17,11 @@ emits the message counts for both uni-directional and bi-directional calls.
 
 See more info on the wire format in `gRPC over HTTP/2 <https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md>`_.
 
+Note by default, the router filter will emit translated HTTP status code stats (from the cluster namespace) for some types
+of gRPC requests. If only setting gRPC specific metrics is preferred, this behavior can be avoided by setting the field
```

Comment:
Pedantically, "for some types of gRPC requests" should possibly read "for gRPC requests that generate some types of local responses (such as upstream timeout, circuit breaker overflow, etc.)."

If I have that right do you mind updating?